eel/src/storage_persister.rs

```diff
@@ -512,12 +535,22 @@ where
         };
 
         // Persist remotely
+        let encrypted_data = match encrypt(&channel_manager.encode(), &self.encryption_key) {
+            Ok(data) => data,
+            Err(e) => {
+                error!("Failed to encrypt the ChannelManager: {}", e.to_string());
+                return Err(std::io::Error::new(
+                    std::io::ErrorKind::Other,
+                    "Failed to encrypt the ChannelManager",
```

Comment:
```suggestion
                    format!("Failed to encrypt the ChannelManager: {}", e.to_string()),
```

Reply:
Applied, but then had to remove the `.to_string()`: a65fb331ee917f8409816426e04a75fe130f1c30